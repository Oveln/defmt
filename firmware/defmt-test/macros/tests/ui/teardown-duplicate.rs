fn main() {}

#[defmt_test_macros::tests]
mod tests {
    #[teardown]
    fn first() {}

    #[teardown]
    fn second() {}
}
