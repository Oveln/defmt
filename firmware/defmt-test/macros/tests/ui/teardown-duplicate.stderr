error: only a single `#[teardown]` function can be defined
 --> tests/ui/teardown-duplicate.rs:9:8
  |
9 |     fn second() {}
  |        ^^^^^^
